govcd/query.go

```diff
@@ -31,6 +31,16 @@ func (vcdCli *VCDClient) Query(params map[string]string) (Results, error) {
 	return getResult(&vcdCli.Client, req)
 }
 
+func (client *Client) QueryWithNotEncodedParams(params map[string]string, notEncodedParams map[string]string) (Results, error) {
```

Comment:
Missing description.
What's the difference between this function and the one a few lines below with the same name but different receiver?
(`func (vcdCli *VCDClient) QueryWithNotEncodedParams(params map[string]string, notEncodedParams map[string]string)` )

Reply:
Yes, for convenience

Reply:
Can one wrap another if it is a convenience function?

Reply:
done